src/content/developers/docs/dapps/index.md

```diff
@@ -48,7 +48,7 @@ A smart contract is code that lives on the Ethereum blockchain and runs exactly
 
 <YouTube id="F50OrwV6Uk8" />
 
-## Dapp tools {#dapp-tools}
+## Tools for creating dapps {#dapp-tools}
```

Comment:
Changed the English name of this section to avoid confusion-- leaving the #dapp-tools tag for consistency